packages/wonder-stuff-ci/src/mobile-release-utils.ts

```diff
@@ -0,0 +1,54 @@
+import execProm from "./exec-prom";
+import { allMobileReleaseTags } from "./mobile-release-git-utils";
+
+/**
+ *
+ * @returns all release tags sorted creation time ascending
+ */
+export const getMobileReleaseTags = async (): Promise<Array<string>> => {
+    const tags = await allMobileReleaseTags();
+    return tags
+        .filter((tag) =>
+            tag.match(/^(ios|android|unified)-(\d+\.\d+\.\d+(-\w*)*)$/i),
+        )
+        .sort(compareVersions);
+};
+
+/**
+ * Compares two versions (of the form <num>.<num>.<num> ) or tags (of the form <tag>-<num>.<num>.<num>)
+ * @param v1
+ * @param v2
+ * @returns int, 1 if v1 > v2, -1 if v1 < v2, 0 if v1 == v2
```

Comment:
**suggestion:** We use the imperative for function names, and try to keep them under 80 chars in width. Longer descriptive info can go underneath. In addition, the params and returns should have type info and descriptions.

```suggestion
 * Compare two versions or tags
 * @param {string} v1 A version of the form `<num>.<num>.<num>` or tag of the form `<tag>-<num>.<num>.<num>`.
 * @param {string} v2 A version of the form `<num>.<num>.<num>` or tag of the form `<tag>-<num>.<num>.<num>`.
 * @returns {number} 1 if v1 > v2, -1 if v1 < v2, 0 if v1 == v2
```